hydra-node/json-schemas/api.yaml

```diff
@@ -225,6 +227,61 @@ channels:
           method: GET
           bindingVersion: '0.1.0'
 
+  /snapshot/latest:
+    servers:
+      - localhost-http
+    publish:
+      description: |
+        Request to adopt latest confirmed snapshot.
+
+        Possible responses of this endpoint are:
+          * 200: OK
+          * 404: when head was never open
+      operationId: sideLoadLatestConfirmedSnapshotRequest
+      message:
+        payload:
+          type: "null"
```

Comment:
Do we need `message` here if `payload` is null?

Reply:
good catch!
apparently we can!